app/views/nsm/review_and_adjusts/show.html.erb

```diff
@@ -46,6 +46,40 @@
         <%= render partial: "nsm/#{scope}/index", locals: { claim:, records:, summary:, pagy: } %>
       </div>
     </div>
+
+    <%
+        type_changed_fields =
```

Comment:
You don't need this. Even if none of the work items that have changed type have changed from one core cost summary group to another (which is what this value indicates), we still want to show the `[*]` footnote, because there will be a `[*]` asterisk in the work items summary table. So the question is not `if type_changed_fields.any?` but rather `if type_changed_records.any?`

Reply:
I've resolved the other convo now that I've stopped misreading the code but I _think_ this one still stands - if you're on the work items page and a work item has changed type but hasn't changed to a different core cost row, `type_changed_fields` will be blank but we _do_ want to show the footnote.

Reply:
Yeah you're right I have this changed queued up.